pkg/restapi/rp/operation/operations.go

```diff
@@ -561,7 +553,40 @@ func (c *Operation) wellKnownConfig(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (c *Operation) getOIDCConfig(profileID string) ([]byte, error) {
+	resp, err := c.sendHTTPRequest(http.MethodGet,
+		fmt.Sprintf("%s/verifier/profiles/%s/well-known/did-config",
+			c.vcsV1URL, profileID), nil, httpContentTypeJSON, "")
+	if err != nil {
+		return nil, fmt.Errorf(fmt.Sprintf("failed to get did config: %s", err.Error()))
+	}
+
+	respBytes, respErr := io.ReadAll(resp.Body)
+	if respErr != nil {
+		return nil, fmt.Errorf(fmt.Sprintf("failed to read did config resp : %s", err.Error()))
+	}
+
+	defer func() {
+		e := resp.Body.Close()
+		if e != nil {
+			logger.Errorf("closing response body failed: %v", e)
```

Comment:
I would suggest changing to `logger.Errorf("closing response body failed: %s", e.Error())`. I think it's better to use a more specific formatting directive whenever possible.